### ISSUE TITLE
Cannot find file with space(s) in name. 

### ISSUE THREAD

**Issue by hoekbrwr** [Tazzios/pdfviewer#32]
I try to display this file which contains spaces in the filename and it just does not display any content.
Are there more limitations like special characters? Can I use underscore?

**Comment by Tazzios** [comment]
I assume you use the  'file=' option? space are replaced in url encoding with '%20'

**Comment by hoekbrwr** [comment]
Yes using file= option. Removing spaces in name works well. I tried surrounding with "", but that did not work too. Are you sure you replaced spaces with %20??? Dit you really apply url encoding? Finding the file with %20 in name might not work well in all cases!Verzonden vanaf Outlook voor Android

**Comment by Tazzios** [comment]
pdfjs expect an URL encoded URL. here I do urlencoding so I would expect it to work.
https://github.com/Tazzios/pdfviewer/blob/f06d4d7edba44b4cfc02c14fe993b39ec3c7d014/content_plugin/pdfviewer.php#L363

I have to test why it not works.

**Comment by hoekbrwr** [comment]
Something goes wrong with encoding!

When I replace space with %20 then it works. 

Deliberate name mismatch works in then not finding file.

Without any space in name it works.

**Comment by joshercartifes** [comment]
Hi all,
I have a similar problem, I removed the spaces from the name, but it shows an empty pdf. I included in the article in Joomla
`{pdfviewer file="full_path_to_pdf"}`. 
I checked the full path to pdf is working in the browser and it shows the PDF. I do not why the embed viewer is not showing anything.
Thanks

**Comment by Tazzios** [comment]
Fixed with #44 in the upcoming release.

**Comment by Tazzios** [comment]
V1.6.0 released.
